### ISSUE TITLE
LiveSurveyRepository crash on json response

### ISSUE THREAD

**Issue by denis-mfm** [ScreebApp/sdk-ios-public#2]
Hi, we are experiencing some crashes with Screeb 1.12.3 with the following error:

```
Screeb/LiveSurveyRepository.swift:156: Fatal error: 'try!' expression unexpectedly raised an error: Swift.DecodingError.keyNotFound(CodingKeys(stringValue: "emoji", intValue: nil), Swift.DecodingError.Context(codingPath: [CodingKeys(stringValue: "payload", intValue: nil), CodingKeys(stringValue: "question", intValue: nil), CodingKeys(stringValue: "callToAction", intValue: nil), CodingKeys(stringValue: "choices", intValue: nil), _JSONKey(stringValue: "Index 0", intValue: 0), CodingKeys(stringValue: "payload", intValue: nil)], debugDescription: "No value associated with key CodingKeys(stringValue: \"emoji\", intValue: nil) (\"emoji\").", underlyingError: nil))
```

Crash stack trace
```
Crashed: app.screeb.websocket
0  libswiftCore.dylib             0x37d7c _assertionFailure(_:_:file:line:flags:) + 312
1  libswiftCore.dylib             0xadeb0 Dictionary.init<A>(_:uniquingKeysWith:) + 646
2  Screeb                         0x2e4b8 $s6Screeb20LiveSurveyRepositoryC18deserializeMessage7messageAA04Jsonb8ResponseF0VSS_tFTf4nd_n + 608
3  Screeb                         0x2c080 $s6Screeb20LiveSurveyRepositoryC9onMessage7messageySS_tF + 256
4  Screeb                         0x4fe00 $s6Screeb0A9WebsocketC10didReceive5event6clienty10Starscream14WebSocketEventO_AG0hI0CtFyycfU_ + 536
5  Screeb                         0x4f8e8 $sIeg_IeyB_TR + 20
6  libdispatch.dylib              0x2460 _dispatch_call_block_and_release + 32
7  libdispatch.dylib              0x3f88 _dispatch_client_callout + 20
8  libdispatch.dylib              0xb640 _dispatch_lane_serial_drain + 672
9  libdispatch.dylib              0xc18c _dispatch_lane_invoke + 384
10 libdispatch.dylib              0x16e10 _dispatch_workloop_worker_thread + 652
11 libsystem_pthread.dylib        0xdf8 _pthread_wqthread + 288
12 libsystem_pthread.dylib        0xb98 start_wqthread + 8
```

Could you help us to solve it?

**Comment by samber** [comment]
Hi @denis-mfm 

Thanks for the bug report.

We will fix this ASAP.

**Comment by samber** [comment]
Hi @denis-mfm and sorry for the delay.

It has been fixed in v1.12.4 :tada:
